src/lsm/tree.zig

```diff
@@ -618,37 +614,33 @@ pub fn TreeType(comptime Table: type, comptime Storage: type, comptime tree_name
             );
         }
 
-        fn compact_io(tree: *Tree) void {
```

Comment:
The docs in this file still make many references to `compact_io`, can you update them accordingly?